Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/ViewSettingsController.java

```diff
@@ -73,28 +74,38 @@
             "org:resource",
             "org:role"
     );
+    
+    private String[] measures = {
+            "case_count",
+            "case_relative",
+            "total",
+            "mean",
+            "median",
+            "max",
+            "min"
+    };
 
     private Map<String, MeasureAggregation> measureAggMap = new HashMap<String, MeasureAggregation>() {
         {
-            put("case", MeasureAggregation.CASES);
-            put("case_relative", MeasureAggregation.CASES);
-            put("total", MeasureAggregation.TOTAL);
-            put("mean", MeasureAggregation.MEAN);
-            put("median", MeasureAggregation.MEDIAN);
-            put("max", MeasureAggregation.MAX);
-            put("min", MeasureAggregation.MIN);
+            put(measures[0], MeasureAggregation.CASES);
+            put(measures[1], MeasureAggregation.CASES);
+            put(measures[2], MeasureAggregation.TOTAL);
+            put(measures[3], MeasureAggregation.MEAN);
+            put(measures[4], MeasureAggregation.MEDIAN);
+            put(measures[5], MeasureAggregation.MAX);
+            put(measures[6], MeasureAggregation.MIN);
         }
     };
     
     private Map<String, MeasureRelation> measureRelationMap = new HashMap<String, MeasureRelation>() {
```

Comment:
If this map is immutable, would it make sense for this code to be moved to MeasureRelation instead?

Reply:
Same comment as above, the decoupling between UI specifics and data objects is important, not to mix them leading to harder to change code.